portal/src/Widget.tsx

```diff
@@ -1,23 +1,80 @@
-import React from "react";
+import React, {
+  useState,
+  useCallback,
+  useLayoutEffect,
+  useRef,
+  useMemo,
+} from "react";
+import { IconButton, DefaultEffects } from "@fluentui/react";
 import cn from "classnames";
-import { DefaultEffects } from "@fluentui/react";
+
 import styles from "./Widget.module.css";
 
-export interface WidgetProps {
+interface WidgetProps {
   className?: string;
+  extendable?: boolean;
```

Comment:
Currently, when `extendable` is empty, it implies the widget is non-extendable. And the `true` or `false` value of `extendable` is the initial value of `extended`? I think it is a bit confusing.